data-assets/rs-virtualization/deploy/rs-vm-rules-policy.yaml

```diff
@@ -0,0 +1,255 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: prometheus-rules
+spec:
+  remediationAction: enforce
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: prometheus-rules-policy
+        spec:
+          remediationAction: inform
+          severity: low
+          namespaceSelector:
+            include:
+              - openshift-monitoring
+          object-templates:
+            - complianceType: mustonlyhave
+              objectDefinition:
+                apiVersion: monitoring.coreos.com/v1
+                kind: PrometheusRule
+                metadata:
+                  name: acm-right-sizing-rules
+                  namespace: openshift-monitoring
+                spec:
+                  groups:
+                    - interval: 5m
+                      name: acm-vm-right-sizing-namespace-5m.rule
+                      rules:
+                        - expr: |
+                            max_over_time(
+                              sum((
+                                    kubevirt_vm_resource_requests{
```

Comment:
Can we update doc and add some high level info on metric used to calculate cpu/memory usage/request